src/Lean/ToLevel.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2023 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Alex Keizer
+-/
+prelude
+import Lean.Expr
+
+/-! # `ToLevel` class
+This module defines `Lean.ToLevel`, which is the `Lean.Level` analogue to `Lean.ToExpr`.
+-/
+
+namespace Lean
+
+universe w
```

Comment:
Given that Lean core makes use of autoImplicit, could you remove this `universe` command and the `universe u v` command later on?